### ISSUE TITLE
Add alternate options for reporting machine crafting rates

### ISSUE THREAD

**Issue by LevFendi** [Factorio-Access/FactorioAccess#157]
## **1. Is your feature request related to a problem? Please describe.**
Different parts of the game use different units to report machine rates. Some machines report production per minute, other machines report moved items per second, and the recipes report seconds per crafting cycle. These are all useful for their own contexts but a player may prefer to use the same units for everything.

## **2. Describe the solution you'd like**
When the preferences menu gets added, add the toggled option for inserters and belts to report rates in minutes or in seconds. For machines allow reporting in minutes or seconds. For recipes in the crafting menu allow reporting crafts per second (at default speed) or seconds per craft. For each case, maybe limit the answer to three decimal points to prevent excess verbosity.

## **3. Describe alternatives you've considered, if any, perhaps including previous discussions**
The alternative is to apply the same rate across the board by default so that everything is explained per minute, using large numbers and decimal places where needed. Giving more options than this sounds like offering the best user experience.

## **4. Additional context**
For reference: For crafting machines, the vanilla game reports only the crafting speed multiplier. For inserters, it reports the arm rotation speed in degrees per second. While this info helps differentiate different machine tiers from each other, it is a lot less helpful than the mod's reports when comparing different types of machines. I speculate that this might be intentional because the developers want the players to feel out and learn to observe the throughput problems rather than mundanely checking a universal item per minute reader.

**Comment by ahicks92** [comment]
sort of half relevant but useful perhaps to have it here: does vanilla show durations of recipes in the crafting menu?  If you have the that the speed of the machine shouldn't matter as long as all involved are the same speed.

**Comment by LevFendi** [comment]
> does vanilla show durations of recipes in the crafting menu? 

Vanilla tells you how many seconds a recipe takes at 1x crafting speed and it does NOT adjust for the actual speed.

**Comment by ahicks92** [comment]
Right, that's what I'm saying.  You'd do the ratio off the crafting menu and as long a the assembly machines all match up that works out for a sighted player.  Which I guess isn't important to doing this so shrug.
